.tekton/basic-component-pipeline.yaml

```diff
@@ -356,6 +356,31 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: roxctl-image-scan
+    params:
+    - name: image
+      value: $(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)
+    - name: output_format
+      value: table
+    - name: rox_image
+      value: quay.io/stackrox-io/roxctl:4.4.2
+    runAfter:
+    - build-container
+    taskRef:
+      # See https://github.com/tektoncd/catalog/tree/main/task/rhacs-image-scan.
+      params:
+      - name: name
+        value: rhacs-image-scan
+      - name: bundle
+        value: gcr.io/tekton-releases/catalog/upstream/rhacs-image-scan@sha256:6be52f9e623e3fecbf2d2b542ac59cc9c6993311306e78651aba98f4e520aa6c
```

Comment:
The fact that the task comes from `gcr.io` raises a question why not `quay.io`? We should be in control about pull limits from Quay unlike other registries (not sure about gcr.io). Going further, I assume the task is built and published there with something different than Konflux (likely GHA).

The idea, as I understand it, of Konflux is to create a secure and trusted build environment. GHA and external CI systems don't provide that expected level of trust. What your PR does is it makes something built in untrusted environment to run and control builds in trusted environments. While the risks is low that this task bundle will go crazy and do bad things, it still exists.

A proper thing would be to first onboard the task builds to Konflux so that the task is trusted, then use it in our pipelines. That's what the violated EC policy actually says, and I don't think it is good to go the path of [adding a policy exclusion](https://github.com/stackrox/stackrox/pull/11192#issuecomment-2124779112).

Reply:
> Do you see Konflux onboarding as a strict requirement for the dogfooding?

No, I don't see it as a strict requirement.

How about quay.io though?

Reply:
The Tekton catalog publishes to gcr.io, we don't control that. We would have to mirror our bundles to Quay.